app/GNSSStatus/Parsing/GNSSData.cs

```diff
@@ -2,18 +2,32 @@
 using System.Text;
 using GNSSStatus.Configuration;
 using GNSSStatus.Utils;
+using YamlDotNet.Serialization;
 
 namespace GNSSStatus.Parsing;
 
 public class GNSSData
 {
+    public readonly List<double> DeltaZCache = new();
     public readonly List<double> DeltaXCache = new();
     public readonly List<double> DeltaYCache = new();
-    public readonly List<double> DeltaZCache = new();
     public readonly List<double> RoverXCache = new();
     public readonly List<double> RoverYCache = new();
     public readonly List<double> RoverZCache = new();
-    public readonly List<GGAData.FixType> FixTypesCache = new();
+    public readonly List<string> RoverUtcTimeCache = new();
+    public readonly List<GGAData.FixType> RoverFixTypeCache = new();
+    public readonly List<int> RoverSatInUseCache = new();
+    public readonly List<float> RoverPDopCache = new();
+    public readonly List<float> RoverVDopCache = new();
+    public readonly List<float> RoverHDopCache = new();
+    public readonly List<float> RoverErrorLatitudeCache = new();
+    public readonly List<float> RoverErrorLongitudeCache = new();
+    public readonly List<float> RoverErrorAltitudeCache = new();
+    public readonly List<double> RoverBaselineCache = new();
```

Comment:
If everything needs to be averaged/median calculated, wrap in struct member (GGA, GSA, GST...) field setters. It's not maintainable to store everything publicly accessible here.